frontend/src/App.tsx

```diff
@@ -1390,6 +1394,10 @@ export class App extends PureComponent<Props, State> {
           sidebarChevronDownshift:
             this.props.hostCommunication.currentState.sidebarChevronDownshift,
           getBaseUriParts: this.getBaseUriParts,
+          noPadding: hasEmbedUrlParamSetToNoPadding(),
```

Comment:
out of curiosity: why we have negations here? Maybe instead of saying what is excluded we can specify what's included?

Reply:
This is by design, the spec says that our params block things not enable them, also the code that can be found today in main branch also works by disabling things, so it's just the most optimal way to go

Reply:
Yes, I understand that the user specifies what has to be excluded. But then we can translate it into what should be included so that instead of writing `!(embedded || noFooter)` we can write`!embedded && footerEnabled` which I think would read a bit better. But as I said, this is a nit.